tests/cupyx_tests/jit_tests/test_raw.py

```diff
@@ -280,7 +280,7 @@ def f(x, m):
             laneId = jit.threadIdx.x & 0x1f
             if laneId < m:
                 x[laneId] = 1
-                jit.syncwarp(mask=m)
+                jit.syncwarp(mask=(1 << m) - 1)
```

Comment:
Good catch...

Reply:
This is a bit problematic, because the behavior when __syncwarp() is used incorrectly, as shown above, varies depending on the GPU type. H100 terminates with an error of illegal instruction, so I noticed this.